infra/terraform/modules/app-server/user_data.sh

```
@@ -0,0 +1,34 @@
+export DEBIAN_FRONTEND=noninteractive
+
+echo "=== Updating system ==="
+yum update -y
+
+echo "=== Installing Docker ==="
+yum install -y docker
+systemctl enable docker
+systemctl start docker
+
+echo "=== Installing Docker Compose ==="
+curl -sL "https://github.com/docker/compose/releases/download/v${docker_compose_version}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Shellcheck (0.11.0)

[warning] docker_compose_version is referenced but not assigned.

(SC2154)